sci-misc/kaldi/kaldi-0_p20151106.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+inherit eutils flag-o-matic
+
+DESCRIPTION="A toolkit for speech recognition"
+HOMEPAGE="http://kaldi-asr.org/"
+SRC_URI="http://gentoo.akreal.net/distfiles/${P}.tar.xz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="doc test threads atlas cuda"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	media-libs/speex
+	virtual/lapack
+	virtual/lapacke
+	>=sci-misc/openfst-1.4.1
+	atlas? ( sci-libs/atlas[threads=] )
+	!atlas? ( sci-libs/openblas[-openmp,-threads] sci-libs/lapack-reference )
+	cuda? ( dev-util/nvidia-cuda-toolkit )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )"
+
+REQUIRED_USE="!atlas? ( !threads )"
+
+# We need write acccess /dev/nvidiactl, /dev/nvidia0 and /dev/nvidia-uvm and the portage
+# user is (usually) not in the video group
+RESTRICT="cuda? ( userpriv )"
```

Comment:
Yes, I've checked it: when `cuda` USE flag is enabled, sandbox is running as root and tests are passing. This is actually needed only when both `cuda` and `test` are enabled, I've updated ebuild.